### ISSUE TITLE
Pre-train Action recognition videoMAE model on UCF101

### ISSUE THREAD

**Issue by Mr-MeerMoazzam** [OpenGVLab/VideoMAEv2#37]
Hi!!
I'm interested in pre-training an action recognition model using videoMAEV2 on the UCF101 dataset. But I am having some difficulties and want assistance from you guys. Please help me. 

Data Preparation: For Pretraining you have written 
```
for video data line:
video_path 0 -1
```
What does mean by 0 and -1???
What are the next steps to pre-train the model on UCF101? What are some good practices that should be followed before going to pre-train the model??

**Comment by congee524** [comment]
-1 means that the file on the video_path is a video. 0 represents the label, but labels are useless in pre-train, so we just place 0 on the position as the placeholder.

you just need to modify the video_path to the path of UCF101 videos.

**Comment by Mr-MeerMoazzam** [comment]
> -1 means that the file on the video_path is a video. 0 represents the label, but labels are useless in pre-train, so we just place 0 on the position as the placeholder.
> 
> you just need to modify the video_path to the path of UCF101 videos.

Thanks for the quick response!! I really appreciate that. I have some more questions about to pretrain the model. What are the next steps after organizing the paths into the CSV??

**Comment by Mr-MeerMoazzam** [comment]
> -1 means that the file on the video_path is a video. 0 is the start_idx, it's meaningless when the file is video, just a placeholder.
> 
> you just need to modify the video_path to the path of UCF101 videos.

You mean we don't need any labels for pretrain the model?

**Comment by congee524** [comment]
Start training. You can see the doc of [pre-train](https://github.com/OpenGVLab/VideoMAEv2/blob/master/docs/PRETRAIN.md) or follow the instructions in [VideoMAE](https://github.com/MCG-NJU/VideoMAE), we use the similar architecture with VideoMAE.

**Comment by congee524** [comment]
> > -1 means that the file on the video_path is a video. 0 is the start_idx, it's meaningless when the file is video, just a placeholder.
> > you just need to modify the video_path to the path of UCF101 videos.
> 
> You mean we don't need any labels for pretrain the model?

Sure. MAE is a self-supervised method

**Comment by Mr-MeerMoazzam** [comment]
I have a machine with the following specifications
![image](https://github.com/OpenGVLab/VideoMAEv2/assets/98279854/b0f36054-95c2-43c5-af0b-64bba14e7d03)
and have RTX 3060. Is the system capable of pretraining on UCF101?

**Comment by congee524** [comment]
I think not. You may need at least 8 GPUs

**Comment by Mr-MeerMoazzam** [comment]
Can I finetune it on the UCF101 with the system specs??

**Comment by congee524** [comment]
you can try it using the distilled vit-small model.

**Comment by Mr-MeerMoazzam** [comment]
Could you please suggest to me which scripts from [Finetunescripts](https://github.com/OpenGVLab/VideoMAEv2/tree/master/scripts/finetune) I should follow to finetune it?

**Comment by congee524** [comment]
Read the [finetune doc](https://github.com/OpenGVLab/VideoMAEv2/blob/master/docs/FINETUNE.md) and write a new one.

**Comment by Mr-MeerMoazzam** [comment]
What does mean by slurm train and Dist train?
